CyLR/Properties/AssemblyInfo.cs

```diff
@@ -32,7 +32,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.3.0")]
-[assembly: AssemblyFileVersion("1.3.3.0")]
+[assembly: AssemblyVersion("1.3.4.0")]
+[assembly: AssemblyFileVersion("1.3.4.0")]
```

Comment:
It doesn't matter for now, but as a convention, it's usually best to do version changes in a separate commit.